src/main/java/seedu/address/model/ProjectsFolder.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.project.Project;
+import seedu.address.model.project.UniqueProjectList;
+
+/**
+ * Wraps all data at the projects level
+ * Duplicates are not allowed (by .isSameProject comparison)
+ */
+public class ProjectsFolder implements ReadOnlyProjectsFolder {
```

Comment:
Why call it ProjectsFolder instead of something like ProjectList?

Reply:
following the structure of addressbook, 

addressbook contains personList contains persons. 

similarly,

ProjectFolder contains projectList contains projects.